### PR TITLE
Adds Maëlle to team page

### PR DESCRIPTION
**Pull request by shikokuchuo**
Please let me know if you'd like any amendments. Thanks.

### CODE REVIEW

File: team.qmd [hunk 1]
```diff
@@ -38,6 +38,22 @@ Charlie Gao is director of Hibiki AI and a co-creator of R-multiverse. He is the
...
+:::: {.columns}
+::: {.column width=20%}
+<a href="https://github.com/maelle" target="_blank">
+  <img src="https://github.com/maelle.png" class="img-fluid rounded-circle">
+</a>
+:::
+::: {.column width=10%}
+:::
+::: {.column width=70%}
+Maelle is a research software engineer, part-time with rOpenSci, notably maintaining its [Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/) guide. She also created the [R-hub blog](https://blog.r-hub.io/) and co-wrote the online book [HTTP testing in R](https://books.ropensci.org/http-testing/).
```
Comment (maelle) [inline team.qmd:52]:
```suggestion
Maëlle is a research software engineer, part-time with rOpenSci, notably maintaining its [Packages: Development, Maintenance, and Peer Review](https://devguide.ropensci.org/) guide. She also created the [R-hub blog](https://blog.r-hub.io/) and co-wrote the online book [HTTP testing in R](https://books.ropensci.org/http-testing/).
```

**Review by maelle — APPROVED** [review]
thank you!!

File: team.qmd (hunk 1 above)
Comment (shikokuchuo) [reply team.qmd]: oops, sorry for that!

**Comment by shikokuchuo** [comment]
Amended, and thanks!

**Comment by maelle** [comment]
Thank _you_

**Pull request merged**